packages/zosfiles/src/methods/copy/Copy.ts

```diff
@@ -57,14 +57,20 @@ export class Copy {
         ImperativeExpect.toBeDefinedAndNonBlank(options["from-dataset"].dsn, "fromDataSetName");
         ImperativeExpect.toBeDefinedAndNonBlank(toDataSetName, "toDataSetName");
 
+        const targetDataSetExists = await this.dataSetExists(session, toDataSetName);
+
+        let newDataSet = false;
+        if(!targetDataSetExists) {
+            newDataSet = true;
```

Comment:
When possible `const` is preferred over `let` 😋 This could be simplified to:
```suggestion
        const newDataSet = !targetDataSetExists;
        if (newDataSet) {
```